bedrock/products/views.py

```diff
@@ -302,7 +302,17 @@ def resource_center_article_view(request, slug):
 @require_safe
 def relay_landing_page(request):
     template_name = "products/relay/landing.html"
-    ftl_files = ["products/relay/landing", "products/relay/shared"]
+    ftl_files = ["products/relay/shared", "products/relay/landing", "products/relay/bundle", "products/relay/matrix", "products/relay/faq"]
+
+    context = {}
+
+    return l10n_utils.render(request, template_name, context, ftl_files=ftl_files)
```

Comment:
I see this is how it's done in other views, but `context` is an optional argument and you can just remove it.

```suggestion
    return l10n_utils.render(request, template_name, ftl_files=ftl_files)
```